src/plugins/workspace/public/components/workspace_detail/workspace_updater.tsx

```diff
@@ -4,30 +4,29 @@
  */
 
 import React, { useCallback, useEffect, useState } from 'react';
-import { EuiPage, EuiPageBody, EuiPageHeader, EuiPageContent } from '@elastic/eui';
+import { EuiPage, EuiPageBody, EuiPageContent, EuiSpacer } from '@elastic/eui';
 import { i18n } from '@osd/i18n';
 import { PublicAppInfo } from 'opensearch-dashboards/public';
 import { useObservable } from 'react-use';
 import { BehaviorSubject, of } from 'rxjs';
 import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
-import { WORKSPACE_OVERVIEW_APP_ID } from '../../../common/constants';
+import { WORKSPACE_DETAIL_APP_ID } from '../../../common/constants';
 import { formatUrlWithWorkspaceId } from '../../../../../core/public/utils';
 import { WorkspaceAttributeWithPermission } from '../../../../../core/types';
 import { WorkspaceClient } from '../../workspace_client';
 import {
-  WorkspaceForm,
   WorkspaceFormSubmitData,
   WorkspaceOperationType,
   convertPermissionsToPermissionSettings,
   convertPermissionSettingsToPermissions,
+  WorkspaceDetailForm,
 } from '../workspace_form';
 import { getDataSourcesList } from '../../utils';
 import { DataSource } from '../../../common/types';
 
 export interface WorkspaceUpdaterProps {
   workspaceConfigurableApps$?: BehaviorSubject<PublicAppInfo[]>;
-  hideTitle?: boolean;
-  maxWidth?: number | string;
+  tab?: string;
```

Comment:
Should be `DetailTab if I understood the props here correctly?

Reply:
Sure, it has been updated.